bindings/java/gateway-java-binding/src/main/java/com/microsoft/azure/gateway/core/IGatewayModule.java

```diff
@@ -7,8 +7,7 @@
 public interface IGatewayModule {
 
     /**
-     * The create method is called by the subclass constructor when the native Gateway creates the module. The constructor
-     * should save the {@code broker} parameter.
```

Comment:
added in the java_binding_hld.md: The implementaion of `IGatewayModule` should save the moduleAddr and broker arguments from `create` method in order to use them later to send messages to gateway.